doc/whatsnew.rst

```diff
@@ -7,6 +7,12 @@ What's new
 v2024.4.22
 ==========
 
+- Made changes to :doc:`/api/tools-costs` (:pull:`186`).
```

Comment:
These lines should go into the "next release" section since they were not actually included in v2024.4.22. To do that, please uncomment lines 4 and 5 (removing the `..`) and start a new list there :)

Reply:
Especially if you anticipate more such changes, you could make this a whole subsection:
```
Changes to :doc:`/api/tools-costs` (:pull:`186`, <complete list later on>)
--------------------------------------------------------------------------

- Fix jumps in ...
```

Reply:
Thanks @glatterf42 ! I've created a subsection for changes in the tools module (I think more changes could be forthcoming) and moved all that to the "next release" section.